src/3.1/entrypoint.sh

```diff
@@ -15,40 +15,39 @@ export LD_LIBRARY_PATH=${MUSIC_PATH}/lib:$LD_LIBRARY_PATH
 export PATH=${MUSIC_PATH}/bin:$PATH
 export CPATH=${MUSIC_PATH}/include:$CPATH
 export PYTHONPATH=${MUSIC_PATH}/lib/python3.8/site-packages:$PYTHONPATH
-export NEST_SERVER_RESTRICTION_OFF=true
-export NEST_SERVER_MODULES=nest,numpy
 
-if [[ ! -d /opt/data ]]; then
-    mkdir /opt/data
-fi
+# Create folder if not existed and then change the directory.
+mkdir_cd () {
+    if [[ ! -d $1 ]]; then
+        mkdir $1
+    fi
+    cd $1
+}
```

Comment:
I'm not sure if it makes sense to factor this out. It is the slow version of `mkdir -p $1; cd $1;`, the `if` just makes it slow and the function adds complexity that the operation does not require. My apologies, I didn't see that during our discussion.
```suggestion
```